samigo/samigo-app/src/webapp/jsf/delivery/discrepancyInData.jsp

```diff
@@ -40,16 +40,15 @@
   <!-- content... -->
   <h3><h:outputText value="#{deliveryMessages.data_discrepancy_title}"/></h3>
   <f:verbatim>&nbsp;</f:verbatim>
+  <div class="alertMessage">
```

Comment:
Please use .sak-banner-warn or .sak-banner-error instead of .alertMessage.

Reply:
Hello @fostersdesign,

I don't see any style classes called **sak-banner-warn** or **.sak-banner-error** anywhere in Samigo or Sakai.  Or did you mean I should create them?

However, there is at least one other place in Samigo that uses **alertMessage**.  I don't see why it can't be used?

But... there is another style _in_ samigo called **messageSamigo2**, that looks similar to an alert.  Should I use _that_ style?

Reply:
These new banner classes were added to the SASS files in SAK-41866 / #6966 for Sakai 20. We're trying to move away from the various other message classes to use these standard new ones as a part of the SWITCH standardization project.

Reply:
sorry about that, I must have done a bad search when I tried to search the code from master.  I'll use .sak-banner-error.

although, it would be nice if this patch could be merged to 19.x... but 19.x doesn't have the patch from SAK-41866 / #6966.  if / when it gets merged to 19.x, would it be ok to use alertMessage _then_?

Reply:
Sorry, I'm not familiar with how we handle different implementations for different versions (e.g. previous version). Maybe @ern or @bjones86 or @ottenhoff might be able to provide some recommendations.

Reply:
If it depends on things that don't exist in `19.x`, they typically won't be merged back to that version.

Alternatively, you could submit a follow up PR against `19.x` with the necessary changes, after this has been merged to `master`, or you could backport to your local fork and make the changes there (obviously not as ideal).

Reply:
I would suggest a custom PR for 19.x explaining why the normal merge process can't be used

Reply:
I'll backport to my local 19.x fork for now.

Also, I've committed the latest changes you guys requested.